src/aleph_client/commands/program.py

```diff
@@ -110,9 +143,24 @@ def upload(
     )
 
     volumes = []
-    for volume in prompt_for_volumes():
-        volumes.append(volume)
-        typer.echo("\n")
+
+    ''' '''
+    if persistent_volume is None or ephemeral_volume is None or immutable_volume is None:
+        for volume in prompt_for_volumes():
+            volumes.append(volume)
+            typer.echo("\n")
+
+        '''else  Parse all the volumes that have passed as the cli parameters and put it into volume list '''
```

Comment:
This is a triple quote string, which is evaluated at runtime. There's two cases to use triple quote strings:
1) To write docstrings at the beginning of functions or classes
2) To have a multi-line string including line breaks

Please use regular comment notation `# this is a comment` in this case.
https://stackoverflow.com/questions/68848991/in-python-3-5-how-are-triple-quotes-considered-comments-by-the-ide